modules/sdk-core/src/bitgo/wallet/wallet.ts

```diff
@@ -2405,7 +2407,7 @@ export class Wallet implements IWallet {
     if (!this.baseCoin.isValidAddress(recipientAddress)) {
       throw new Error(`Invalid recipient address ${recipientAddress}`);
     }
-    const baseAddress = this.coinSpecific()?.baseAddress;
+    const baseAddress = this.coinSpecific()?.baseAddress || this.coinSpecific()?.rootAddress;
```

Comment:
can we do `this.baseCoin.getBaseAddress` ?
abstract the coinSpecific things

Reply:
<img width="784" alt="image" src="https://github.com/user-attachments/assets/a931054d-9ea6-48dd-ab00-98a1f12343b3" />